infra/frontend/service/main.tf

```diff
@@ -104,9 +105,7 @@ module "service" {
   vpc_id                = data.aws_vpc.default.id
   subnet_ids            = data.aws_subnets.default.ids
   enable_autoscaling    = module.app_config.enable_autoscaling
-  # TODO: Reroute SSL to production environment
-  # To temporarily avoid issues in production & Terratest: Check if environment is "dev" and workspace is "default"
-  cert_arn = var.environment_name == "dev" && terraform.workspace == "default" ? data.aws_acm_certificate.frontend_cert.arn : null
+  cert_arn              = var.domain != null ? data.aws_acm_certificate.cert[0].arn : null
```

Comment:
Super nit is maybe to check the truthiness of the cert here cause that makes more sense kinda maybe, but achieves the same thing so all good entirely.

`count.data.aws_acm_certificate.cert > 0`